src/screens/ProfileScreen.tsx

```diff
@@ -1,64 +1,161 @@
+      },
+    },
+  ];
 
-  return (
-    <View style={styles.container}>
-      <StyledText style={styles.huge}>Profile Page</StyledText>
-      <WhiteSpace />
-      <StyledText style={styles.huge}>{`Hi ${user.displayName}!`}</StyledText>
-      <WhiteSpace />
+  // Helper sub-component to render the info sections of the profile screen
+  const InfoSection = ({ text, subtext }: IInfoSectionProps) =>
+    useMemo(
+      () => (
+        <View style={styles.infoContainer}>
+          <StyledText fontWeight="bold">{text}</StyledText>
+          <StyledText style={theme.styles.subText}>{subtext}</StyledText>
+        </View>
+      ),
+      [text, subtext]
+    );
 
-      <Button
-        type={"primary"}
-        onPress={() => navigation.navigate(ROUTES.SUGGEST_POI)}
-      >
-        <StyledText>Go to POI Suggestion</StyledText>
-      </Button>
-      <WhiteSpace />
-      <Button
-        type={"primary"}
-        onPress={() => navigation.navigate(ROUTES.NOTIFICATIONS)}
-      >
-        <StyledText>Go to notifications</StyledText>
-      </Button>
-      <WhiteSpace />
-      <Button
-        type={"primary"}
-        onPress={() => navigation.navigate(ROUTES.THEME)}
+  return (
+    <ScrollView style={[theme.styles.screenContainer, styles.container]}>
+      <View style={styles.header}>
+        <Image
+          source={
+            user?.photoURL
+              ? { uri: user.photoURL }
+              : require("@assets/images/avatar.png")
+          }
+          style={styles.profileImg}
+        />
+        <View style={styles.profileInfo}>
+          <StyledText fontWeight="bold" style={styles.name}>
+            {user!.displayName}
+          </StyledText>
+          <StyledText style={styles.email}>{user!.email}</StyledText>
+          <StyledText style={styles.points}>1271 points</StyledText>
+        </View>
+      </View>
+      <List>
+        <List.Item>
+          <InfoSection
+            text="Most frequented POI"
+            subtext="Location 6 • 0.1 km away"
+          />
+        </List.Item>
+        <List.Item>
+          <InfoSection
+            text="Time spent waiting in line"
+            subtext="7 hrs 2 min"
+          />
+        </List.Item>
+      </List>
+      <List
+        styles={{
+          Body: { borderTopWidth: 0 },
+          BodyBottomLine: { borderBottomWidth: 0 },
+        }}
       >
-        <StyledText>Go to appearance settings</StyledText>
-      </Button>
-    </View>
+        {navigationOptions.map(({ name, onPress, style }) => (
+          <List.Item
+            key={name}
+            style={styles.navigationOption}
+            arrow="horizontal"
+            onPress={onPress}
+          >
+            <StyledText
+              style={[styles.navigationText, style]}
+              fontWeight="bold"
+            >
+              {name}
+            </StyledText>
+          </List.Item>
+        ))}
+      </List>
+    </ScrollView>
   );
 };
 
 const styles = StyleSheet.create({
```

Comment:
I'd like to figure out a set of standard styles we can use once we're done prototyping to clean this up a little bit. Maybe that's something Rahil can help with.